SketchUp/Sketchup/ComponentDefinition.rb

```diff
@@ -436,7 +427,7 @@ def name=(name)
   #   componentdefinition = definitions.load path
   #   path = componentdefinition.path
   #
-  # @return path - a valid path if successful, false if
+  # @return [String, FalseClass] a valid path if successful, false if
```

Comment:
Use `true`, `false` and `nil` instead of `TrueClass`, `FalseClass` and `NilClass`.